__tests__/subscription.with-custom-credentials.test.ts

```diff
@@ -1,6 +1,8 @@
-import PubSubService from '../src/service/pubsub';
-import { Subscribers, SubscriberTuple } from '../src/subscriber';
-import { SubscriptionService } from '../src';
+import {
+  PubSubService,
+  SubscriptionService,
+  SubscriberTuple,
```

Comment:
Codacy has a fix for the issue: [Replace `berTupl` with `ptionServic`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8106467)
```suggestion
  SubscriptionService,
```